app/components/ui/Breadcrumbs.vue

```
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+const { breadcrumbs, jsonLd } = useBreadcrumbs()
```

ESLint

[error] 'useBreadcrumbs' is not defined.

(no-undef)